src/components/fidelity_bond/FidelityBondDetailsSetupForm.tsx

```diff
@@ -0,0 +1,582 @@
+  utxos: Utxos
+  onSelected: (utxos: Utxos) => void
+}
+const SelectUtxosStep = ({ balanceSummary, account, utxos, onSelected }: SelectUtxosStepProps) => {
+  const settings = useSettings()
+  const [selectedUtxos, setSelectedUtxos] = useState<Utxos>([])
+
+  const selectedUtxosAmountSum = useMemo(
+    () => selectedUtxos.reduce((acc, current) => acc + current.value, 0),
+    [selectedUtxos]
+  )
+
+  useEffect(() => {
+    setSelectedUtxos([])
+  }, [utxos])
+
+  useEffect(() => {
+    onSelected(selectedUtxos)
+  }, [selectedUtxos, onSelected])
+
+  // TODO: add `calculatedTotalBalanceInSats`
+  const walletTotalBalanceInSats = useMemo(
+    () => balanceSummary.calculatedAvailableBalanceInSats + balanceSummary.calculatedFrozenOrLockedBalanceInSats,
+    [balanceSummary]
+  )
+
+  const relativeSizeToTotalBalance = useMemo(() => {
+    if (walletTotalBalanceInSats <= 0) return 0
+    return selectedUtxosAmountSum / walletTotalBalanceInSats
+  }, [selectedUtxosAmountSum, walletTotalBalanceInSats])
+
+  return (
+    <>
+      <h2>Select UTXOs</h2>
+
+      <rb.Card
+        className="w-100"
+        style={{
+          position: 'sticky',
+          top: '75px',
+          zIndex: 10,
+        }}
+      >
+        <rb.Card.Body style={{ padding: '0.25rem' }}>
+          <rb.Table
+            className="rounded"
+            variant={settings.theme}
+            style={{
+              marginBottom: 0,
+            }}
+          >
+            <tbody>
+              <tr>
+                <td>Account</td>
+                <td className="text-end">#{account.account}</td>
+              </tr>
+              <tr>
+                <td>Total sum of selected UTXOs</td>
+                <td className="text-end">
+                  <Balance
+                    valueString={`${selectedUtxosAmountSum}`}
+                    convertToUnit={settings.unit}
+                    showBalance={settings.showBalance}
+                  />
+                </td>
+              </tr>
+              <tr>
+                <td className="border-0">Relative size to your total balance</td>
+                <td className="border-0 text-end">{(relativeSizeToTotalBalance * 100).toFixed(2)}%</td>
+              </tr>
+            </tbody>
+          </rb.Table>
+        </rb.Card.Body>
+      </rb.Card>
+
+      <div className="mt-3">
+        <UtxoSelector utxos={utxos} onChange={setSelectedUtxos} />
+      </div>
+    </>
+  )
+}
+
+interface SelectLockdateStepProps {
+  utxos: Utxos
+  onChange: (lockdate: Api.Lockdate) => void
+}
+const SelectLockdateStep = ({ utxos, onChange }: SelectLockdateStepProps) => {
+  const { i18n } = useTranslation()
+  const settings = useSettings()
+
+  const [lockdate, setLockdate] = useState<Api.Lockdate | null>(null)
+  const yearsRange = useMemo(() => {
+    if (process.env.NODE_ENV === 'development') {
+      // allow creating lockdate in the past in dev mode
+      return toYearsRange(-1, DEFAULT_MAX_TIMELOCK_YEARS)
```

Comment:
That's what I meant above. Do we still need the development mode now assuming we also add an Advanced collapsible showing the address (logging to console for now is fine as well). 😄

Reply:
Yes. Gonna remove `DevOnly` component. :+1: